edge/pkg/edged/edged.go

```diff
@@ -27,6 +27,8 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"k8s.io/kubernetes/pkg/volume/local"
+	"k8s.io/kubernetes/pkg/volume/nfs"
```

Comment:
Please sort the imports :)

Reply:
Resolved and tested `make verify-vendor`